admin/base/settings/defaults.py

```diff
@@ -10,6 +10,11 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
 
+# from the OSF settings
+BUILT_TEMPLATES = osf_settings.BUILT_TEMPLATES
```

Comment:
I don't really know how this stuff works but it appears that the two template variables refer to mako templates and aren't used anywhere. Since the admin project is strictly django and not flask, are these necessary?